docker-compose.yml

```
@@ -22,7 +22,7 @@ services:
     networks:
       wg:
         ipv4_address: 10.42.42.42
-        ipv6_address: fd00::2a
+        ipv6_address: fdcc:ad94:bacf:61a4::2a
```

Also changing the subnet here at this occasion to match the new prefix:
```suggestion
        ipv6_address: fdcc:ad94:bacf:61a3::2a
```